bin/gwu/gwuLib.ml

```diff
@@ -1533,6 +1533,8 @@ let gwu opts isolated base in_dir out_dir src_oc_ht (per_sel, fam_sel) =
   let out_oc_first = ref true in
   let _ofile, oc, close = opts.oc in
   let origin_file fname =
+    let fname = Filename.basename fname in
+    let fname = if fname = "." then "new_data.gw" else fname in
```

Comment:
why would `fname = "."`?
if there is multiple origin_file with `fname = "."`, a previous `new_data.gw` will be erased.
also this will silently replace a already existing `new_data.gw` file (from a previous use of gwu)

Reply:
I did this fix a bit blindly, after some trials.
The "." case appears when there is no origin_file recorded in the base. I am not sure why it is not "".
If I dont do anything, I get: 
```
(base) Henri@iMac-H GeneWeb-Bases % ../../GitHub/hgouraud/geneweb/distribution/gw/gwu -odir sources2 HenriT1
encoding: utf-8
gwplus
############################################################Uncaught exception: Sys_error("sources2/.: Is a directory")
(base) Henri@iMac-H GeneWeb-Bases % 
```
Testing for this "." case removes the error, and the behavior is otherwise consistent.

My understanding of the mechanism is the following:
* at creation time (gwc):
  - the name of the file from which the data was extracted is recorded.
  - when creating a new family, origin_file is inferred from other marriages of the spouses
  - if this is a brand new family, origin_file is empty

* at save time (gwu), the following happens:
  - if no -o parameter is supplied, the output is directed to `stdout`
  - if `-o file.gw` is provided, the output is directed to `file.gw`
  - if `-odir dir` is provided, the output is directed to `dir/origin_file` when origin_file is not empty, and to `stdout` when origin_file is empty (or ="." ??)

Creating this `new_data.gw` file is unnecessary and can be managed through stdout (`gwu ... > new_data.gw`)
There is a need to create dir (in `-odir dir`) if it does not exist!!